### PR TITLE
Minor refactoring for better readability

### PR DESCRIPTION
**Pull request by vsantwana**

### CODE REVIEW

File: src/_posts/2021-11-18-Visualizing_my_browsing_history.org [hunk 1]
```diff
@@ -25,10 +25,10 @@ Are there sites I always "end up" on?
 ------
 
 While the original idea - of tracking flow from URL to URL - is a good idea,
-it's not a good visualization. The graph would have 120,000+ nodes. 
-And that's a lot of nodes.
+it's not a good visualization. To give you an idea, the graph for my history would have 120,000+ nodes. 
+And thats A LOT OF NODES!
```
Comment (milindl) [inline src/_posts/2021-11-18-Visualizing_my_browsing_history.org]: Grammar error, should be `that's`.

File: src/_posts/2021-11-18-Visualizing_my_browsing_history.org [hunk 2]
```diff
@@ -40,7 +40,7 @@ database - ~places.sqlite~. It's made per-profile, and for my Windows machine,
 it was located at 
 ~C:\Users\<username>\AppData\Roaming\Mozilla\Firefox\Profiles\<profile name>\places.sqlite~.
 
-To avoid any accidents, I made a copy and worked on that.
+To keep my browing history database safe from any corruptions, I made a copy to work on:
```
Comment (milindl) [inline src/_posts/2021-11-18-Visualizing_my_browsing_history.org]: s/corruptions/corruption

**Review by milindl — APPROVED** [review]
Small changes, please make

File: src/_posts/2021-11-18-Visualizing_my_browsing_history.org (hunk 1 above)
Comment (vsantwana) [reply src/_posts/2021-11-18-Visualizing_my_browsing_history.org]: done

**Comment by vsantwana** [comment]
@milindl I have addressed the review comments. Please merge.

**Pull request merged**